TI_Practice1/Validator.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+
+namespace Practice1
+{
+    class Validator
+    {
+        private int _attempts;
+
+        public Validator(int attempts)
+        {
+            _attempts = attempts;
+        }
+
+        public double IsNumber()
+        {
+            int matchCount = 1;
+            double val = 0;
+            while (matchCount <= _attempts)
+            {
+                var input = Console.ReadLine();
+                input = input.Replace(',', '.');
```

Comment:
I can't enter any decimal value for now

![image](https://user-images.githubusercontent.com/9382934/109702609-1fe33d00-7b9d-11eb-959c-5b4720a1444b.png)

Reply:
Not able to reproduce using EN/RU keyboards http://prntscr.com/10bmxsg

Reply:
it depends on the locale of the machine.
Actually you can fix this issue = make your program understand all the locales, but I suggest doing it only if you have time in future.